.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: docker release
+
+on:
+  release:
+    types: [released,prereleased,edited]
+
+env:
+  LATEST_RELEASE: 0.6
+
+jobs:
+  test:
```

Comment:
(nitrpick)
The `job` name is `test`, but it doesn't look like this job tests anything. It could better to name it somehow differently. Like `tagging` or something else.

Reply:
good idea